### PR TITLE
docs: use ASCII characters

### PR DESCRIPTION
**Pull request by sobrinho**
Closes: #1917

- [x] Docs
- [ ] Tests

Screen Shot 2022-02-27 at 18 52 42

### CODE REVIEW

**Comment by sobrinho** [comment]
I believe I replaced every char to a ascii equivalent.

If there are more, let me know!

The script I used is at the first commit message.

File: docs/api/conventions.md [hunk 1]
```diff
@@ -187,8 +187,8 @@ Please note that you can use either `.jsx` or `.tsx` file extensions depending o
 <!-- prettier-ignore -->
 ```markdown [3]
 app/
-├── routes/
-└── root.tsx
```
Comment (kentcdodds) [inline docs/api/conventions.md:191]: Does this look bad on non-macs? I think these should be fine as-is... It's just the table thing we had that should be changed I think.

Comment (kiliman) [reply docs/api/conventions.md:191]: Windows uses the same characters, so should be fine.
```
Volume serial number is EE88-C117
C:\PROJECTS\SCRATCH\MY-REMIX-APP\APP
│   entry.client.tsx
│   entry.server.tsx
│   root.tsx
│
└───routes
        index.tsx
```

**Comment by MichaelDeBoey** [comment]
@sobrinho As @kentcdodds & @kiliman pointed out, you should only change the tables, not the file trees.

Please rebase your branch on latest `main` & revert changes to anything other than the tables.

**Comment by sobrinho** [comment]
Done but that way the scripts won't be automatic if you try again since they will change all the chars as I wrote it.

**Review by kentcdodds — APPROVED** [review]
Thanks for taking the time to do this 👍

**Pull request merged**